src/main_menu.cpp

```diff
@@ -785,20 +786,24 @@ bool main_menu::load_character_tab()
                 } );
                 savegames.erase( new_end, savegames.end() );
             }
+
+            mvwprintz( w_open, iMenuOffsetY - 2 - sel2, 15 + iMenuOffsetX + extra_w / 2, h_white, wn.c_str() );
```

Comment:
All those magical constants in position arguments make it hard to understand where do things go.
The whole thing would be much more readable if that `15`, `40` and similar had names. Say, `worldmenu_x_offset` or something like that.

Reply:
This whole thing could use a refactor for sure.